ui/drivers/qt/ui_qt_themes.h

```diff
@@ -24,13 +24,16 @@ static const QString qt_theme_dark_stylesheet = QStringLiteral(R"(
       background-color:rgb(53,53,53);
       selection-background-color:%1;
    }
-   QWidget#playlistWidget, QWidget#browserWidget, QWidget#tableWidget, QWidget#logWidget {
+   QWidget#playlistWidget, QWidget#browserWidget, QStackedWidget#centralWidget, QWidget#logWidget {
       background-color:rgb(66,66,66);
       border-top:1px solid rgba(175,175,175,50%);
       border-left:1px solid rgba(125,125,125,50%);
       border-right:1px solid rgba(125,125,125,50%);
       border-bottom:1px solid rgba(25,25,25,75%);
    }
+   QStackedWidget#centralWidget {
+      padding: 10px;
```

Comment:
It should be 8px to be similar to the other borders.

Reply:
I changed the widgets that have a border to QFrame so that their follow [the box model](http://doc.qt.io/qt-5/stylesheet-customizing.html#the-box-model) and so the border can be changed in the stylesheet.
I also moved the playlist icon size there. My concern is that it may look too big on some screens so I don't know, maybe we should remove it.
In any case, it can be changed at runtime by loading a stylesheet.